src/tests/bdd/document/reference.feature

```diff
@@ -54,7 +54,7 @@ Feature: Add and remove references
       "isRoot": true
     }
     """
-    Given i access the resource url "/api/reference/test-DS/$1.content.0"
+    Given i access the resource url "/api/reference/test-DS/$1.content[0]"
```

Comment:
This might cause some issues in frontend, but it is more consistent with the other API endpoints